.buildkite/steps/controller.sh

```diff
@@ -2,19 +2,24 @@
 
 set -Eeufo pipefail
 
+echo --- Installing packages
 apt update && apt install -y --no-install-recommends jq
 
+
+echo --- Installing ko
+KO_VERSION=0.13.0
 OS=$(go env GOOS)
-ARCH=$(go env GOARCH)
-if [ "$ARCH" == "amd64" ]; then
-    ARCH="x86_64"
-fi
-latest=$(curl -L -s https://api.github.com/repos/ko-build/ko/releases/latest | jq -r '.tag_name')
-curl -sSfL "https://github.com/ko-build/ko/releases/download/${latest}/ko_${latest:1}_${OS^}_${ARCH}.tar.gz" | tar -xzv -C /bin ko
+ARCH=$(uname -m)
+curl -sSfL "https://github.com/ko-build/ko/releases/download/v${KO_VERSION}/ko_${KO_VERSION}_${OS^}_${ARCH}.tar.gz" | tar -xzv -C /bin ko
```

Comment:
i hadn't heard of [ko](https://github.com/ko-build/ko) before, cool stuff

Reply:
Could be nice to have in the agent pipeline... But we make a few different docker images and this does not have that flexibility.